docs/index.html

```diff
@@ -53,11 +53,11 @@ <h2>xrpl</h2></div>
 </ol>
 <a id="md:case-by-case-setup-steps" class="tsd-anchor"></a><h3><a href="#md:case-by-case-setup-steps">Case by Case Setup Steps</a></h3><p>If you&#39;re using xrpl.js with React or Deno, you&#39;ll need to do a couple extra steps to set it up:</p>
 <ul>
-<li><a href="./UNIQUE_SETUPS.md#using-xrpljs-from-a-cdn">Using xrpl.js with a CDN</a></li>
-<li><a href="./UNIQUE_SETUPS.md#using-xrpljs-with-create-react-app">Using xrpl.js with <code>create-react-app</code></a></li>
-<li><a href="./UNIQUE_SETUPS.md#using-xrpljs-with-react-native">Using xrpl.js with <code>React Native</code></a></li>
-<li><a href="./UNIQUE_SETUPS.md#using-xrpljs-with-vite-react">Using xrpl.js with <code>Vite React</code></a></li>
-<li><a href="./UNIQUE_SETUPS.md#using-xrpljs-with-deno">Using xrpl.js with <code>Deno</code></a></li>
+<li><a href="https://github.com/XRPLF/xrpl.js/blob/main/UNIQUE_SETUPS.md#using-xrpljs-from-a-cdn">Using xrpl.js with a CDN</a></li>
```

Comment:
Is there a way in typedoc to populate urls?  This approach will not link properly for branches.

Reply:
the problem is that the docs don't generate separate pages for github pages, I'll look into the typedoc again and find a solution